### PR TITLE
chore: create config.yml for Welcome Bot

### LINKED ISSUE #717: Create the config.yml file for the Welcome Bot.
## Description
As a first time contributor,
I need a bot to help guide my first contribution, 
so that I can follow this org's contribution guidelines.

## Mocks
Screen Shot 2020-08-06 at 10 26 14 AM

### Update [Required]
- [ ] newIssueWelcomeComment is complete with the correct links to this repo.
- [ ] newPRWelcomeComment is complete with the links to the project's stream or topic and the repo's Code of Conduct.
- [ ] firstPRMergeComment is complete.

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by mayburgos**
### Description

Welcome bot was finally activated for all repos. All of the messages in the config file are updated to mentorship backend's links as well as Zulip's mentorship-backend topic.

Fixes #717 

### Type of Change:
- Outreach
- Documentation

### How Has This Been Tested?

@Tlazypanda has run tests on her own [repo](https://github.com/Tlazypanda/testing_bots).

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: |
+  Hello there!👋 Welcome to the project!💖
+  Thank you and congrats 🎉 for opening your first PR on this project.✨
+  Please ensure your Pull Request meets the [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide).
+  We will review it soon! Till then you can checkout the `README.md` for more details on it.
+
+  Moja-Global is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code Of Conduct]().
```
Comment (SanketDG) [inline .github/config.yml]: Will this be changed?

Comment (isabelcosta) [reply .github/config.yml]: Very nice catch @SanketDG 👏

Comment (meenakshi-dhanani) [reply .github/config.yml]: We need to remove the Moja Global text right? @mayburgos

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  Hello there!👋 Welcome to the project!💖
+
+  Thank you and congrats 🎉 for opening your very first issue in this project.
+  AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code Of Conduct](https://github.com/anitab-org/mentorship-backend/blob/develop/.github/code_of_conduct.md).🙌🏻
+
+  You may submit a PR on the same if you like!
+  Please follow our [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide) while sending PRs.
+  If you want to report a bug🐞, please follow our [bug_report.md](https://github.com/anitab-org/mentorship-backend/blob/develop/.github/ISSUE_TEMPLATE/bug_report.md) template.
+  Also please make sure you include steps to reproduce it and be patient we will get back to you as soon as we can.👀
+
```
Comment (meenakshi-dhanani) [inline .github/config.yml:16]: Should we also add the link to the calendar? For contributors to join project specific open sesssions eg. Mentorship Open Session?

**Review by som23ya — DISMISSED** [review]

File: .github/config.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+firstPRMergeComment: |
+  Wohoo!🎊 Congrats on merging your first contribution to Moja-Global!✨
+  Your code is officially a part of Moja-Global now!! Time for a celebration 🙌🎉⚡️
+  Feel free to explore more on this project.🤗
+  Join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/).🤝 We have different streams for each active repo's for discussions.👩🏻‍💻🧑🏻‍💻
+  Do join our weekly check-in stream to share your weekly goals and the awesome work you did here.😃
+
+  <details>
+  <summary><b>Now that you've completed this, you can help someone else take their first step!</b></summary>
+  <p>Check out projects stream on Zulip and the `first-timer-only` issues to check if someone need help to figure out their problem or need some feedbacks.Reach out and be encouraging and friendly! 😄 🎉</p>
```
Comment (isabelcosta) [inline .github/config.yml]:
```suggestion
  <p>Check out projects stream on Zulip and the `First Timers Only` issues to check if someone needs help to figure out their problem or need some feedbacks. Reach out and be encouraging and friendly! 😄 🎉</p>
```

Comment (isabelcosta) [reply .github/config.yml]: Give me 1 day, and I will review this @mayburgos

File: .github/config.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  Hello there!👋 Welcome to the project!💖
+
+  Thank you and congrats 🎉 for opening your very first issue in this project.
+  AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code Of Conduct](https://github.com/anitab-org/mentorship-backend/blob/develop/.github/code_of_conduct.md).🙌🏻
```
Comment (isabelcosta) [inline .github/config.yml]:
```suggestion
  AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code Of Conduct](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/code_of_conduct.md).🙌🏻
```

File: .github/config.yml (hunk 2 above)
Comment (isabelcosta) [reply .github/config.yml:16]: That is interesting, here is the code for it, if @mayburgos  wants to put it
```
[PST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/Los_Angeles) || [EST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=America/New_York) || [GMT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=GMT) || [WAT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Africa/Lagos) || [IST](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Asia/Colombo) || [AEDT](https://calendar.google.com/calendar/embed?src=sh10tv3mtfve62somg9nngp9tg%40group.calendar.google.com&ctz=Australia/Sydney)
```

File: .github/config.yml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  Hope you have a great time here :)
+
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: |
+  Wohoo!🎊 Congrats on merging your first contribution to Moja-Global!✨
+  Your code is officially a part of Moja-Global now!! Time for a celebration 🙌🎉⚡️
+  Feel free to explore more on this project.🤗
+  Join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/).🤝 We have different streams for each active repo's for discussions.👩🏻‍💻🧑🏻‍💻
```
Comment (isabelcosta) [inline .github/config.yml]:
```suggestion
  Join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). 🤝 
  We have different streams for each active repository for discussions. 👩🏽‍💻🧑🏼‍💻
``` 

If we are using people emojis I will suggest mixing them a bit 🙃

File: .github/config.yml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: |
+  Hello there!👋 Welcome to the project!💖
+  Thank you and congrats 🎉 for opening your first PR on this project.✨
+  Please ensure your Pull Request meets the [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide).
+  We will review it soon! Till then you can checkout the `README.md` for more details on it.
+
+  Moja-Global is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code Of Conduct]().
+
+  Incase you want to join us on [Moja-Global Open Source Slack]() , please drop an email [here](mailto:)
```
Comment (isabelcosta) [inline .github/config.yml]:
```suggestion
  Feel free to join our community at [AnitaB.org Open Source Zulip Community](https://anitab-org.zulipchat.com/) to be kept up to date with our projects discussions and community activities.
```

File: .github/config.yml [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+  For mentorship-backend, join [#**mentorship-system>mentorship-backend**](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system/topic/mentorship-backend)! Hope you have a great time here :)
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: |
+  Hello there!👋 Welcome to the project!💖
+  Thank you and congrats 🎉 for opening your first PR on this project.✨
+  Please ensure your Pull Request meets the [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide).
+  We will review it soon! Till then you can checkout the `README.md` for more details on it.
```
Comment (isabelcosta) [inline .github/config.yml]:
```suggestion
  We will review it soon! Until then you can check out the [`README.md`](../README.md) for more details on it.
```

File: .github/config.yml [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  Hello there!👋 Welcome to the project!💖
+
+  Thank you and congrats 🎉 for opening your very first issue in this project.
+  AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code Of Conduct](https://github.com/anitab-org/mentorship-backend/blob/develop/.github/code_of_conduct.md).🙌🏻
+
+  You may submit a PR on the same if you like!
+  Please follow our [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide) while sending PRs.
```
Comment (isabelcosta) [inline .github/config.yml]:
```suggestion
  Please follow our [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide) and [Coding Standards](../docs/coding_standards.md) while sending PRs.
```

File: .github/config.yml [hunk 9]
```diff
@@ -0,0 +1,53 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  Hello there!👋 Welcome to the project!💖
+
+  Thank you and congrats 🎉 for opening your very first issue in this project.
```
Comment (isabelcosta) [inline .github/config.yml:9]: somewhere in this message should be a appeal to read contribution guidelines

**Comment by vj-codes** [comment]
@mayburgos @isabelcosta hey any specific update for the changes? The welcome bot is implemented on almost all repos except MS backend and android. The anroid PR is only left to merge  so backend lagging behind. Or is there anything else that can be done?

**Comment by vj-codes** [comment]
@isabelcosta hey pinging again what shall be done with this PR ?

File: .github/config.yml [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: |
+  Wohoo!🎊 Congrats on merging your first contribution to Moja-Global!✨
+  Your code is officially a part of Moja-Global now!! Time for a celebration 🙌🎉⚡️
+  Feel free to explore more on this project.🤗
+  Join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). 🤝 
+  We have different streams for each active repository for discussions. 👩🏽‍💻🧑🏼‍💻
+  Do join our weekly check-in stream to share your weekly goals and the awesome work you did here.😃
```
Comment (isabelcosta) [inline .github/config.yml]:
```suggestion
  Woohoo!🎊 Congrats on merging your first contribution to [mentorship-backend](https://github.com/anitab-org/mentorship-backend)!✨
  Your code is officially a part of AnitaB.org Open Source now!! Time for a celebration 🙌🎉⚡️
  Feel free to explore more on this project.🤗
  Join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). 🤝 
  We have different streams for each active repository for discussions. 👩🏽‍💻🧑🏼‍💻
  Do join our weekly check-in stream to share your weekly goals and the awesome work you did here.😃
```

**Comment by vj-codes** [comment]
@isabelcosta the link in bug_report.md also needs to be updated

File: .github/config.yml [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: |
+  Hello there!👋 Welcome to the project!💖
+  Thank you and congrats 🎉 for opening your first PR on this project.✨
+  Please ensure your Pull Request meets the [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide).
+  We will review it soon! Until then you can check out the [`README.md`](../README.md) for more details on it.
+
+  Moja-Global is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code Of Conduct]().
```
Comment (isabelcosta) [inline .github/config.yml]:
```suggestion
  AnitaB.org Open Source is an inclusive community, committed to creating a safe and positive environment 🌸 Please follow our [Code Of Conduct](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/code_of_conduct.md).
```

File: .github/config.yml [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+  AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code Of Conduct](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/code_of_conduct.md).🙌🏻
+
+  You may submit a PR on the same if you like!
+  Please follow our [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide) and [Coding Standards](../docs/coding_standards.md) while sending PRs.
+  If you want to report a bug🐞, please follow our [bug_report.md](https://github.com/anitab-org/mentorship-backend/blob/develop/.github/ISSUE_TEMPLATE/bug_report.md) template.
+  Also please make sure you include steps to reproduce it and be patient we will get back to you as soon as we can.👀
+
+
+  Also feel free to join us on [AnitaB.org Open Source Zulip Community](https://anitab-org.zulipchat.com/).🤝 We have different streams for each active repo's for discussions.👩🏻‍💻🧑🏻‍💻
+  For mentorship-backend, join [#**mentorship-system>mentorship-backend**](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system/topic/mentorship-backend)! Hope you have a great time here :)
```
Comment (isabelcosta) [inline .github/config.yml]:
```suggestion
  Also, feel free to join us on [AnitaB.org Open Source Zulip Community](https://anitab-org.zulipchat.com/) 🤝 We have different streams for each active repository for discussions.👩🏻‍💻🧑🏻‍💻
  For mentorship-backend, join [#mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system) stream! Hope you have a great time here :)
```

File: .github/config.yml [hunk 13]
```diff
@@ -0,0 +1,54 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  Hello there!👋 Welcome to the project!💖
+
+  Thank you and congrats 🎉 for opening your very first issue in this project.
+  AnitaB.org is an inclusive community, committed to creating a safe and positive environment.🌸 Please adhere to our [Code Of Conduct](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/code_of_conduct.md).🙌🏻
```
Comment (isabelcosta) [inline .github/config.yml]:
```suggestion
  AnitaB.org Open Source is an inclusive community, committed to creating a safe and positive environment 🌸 Please follow our [Code Of Conduct](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/code_of_conduct.md) 🙌
```

File: .github/config.yml [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+newIssueWelcomeComment: |
+  Hello there!👋 Welcome to the project!💖
+
+  Thank you and congrats 🎉 for opening your very first issue in this project.
+  AnitaB.org Open Source is an inclusive community, committed to creating a safe and positive environment 🌸 Please follow our [Code Of Conduct](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/code_of_conduct.md) 🙌
+
+  You may submit a PR on the same if you like!
+  Please follow our [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide) and [Coding Standards](../docs/coding_standards.md) while sending PRs.
+  If you want to report a bug🐞, please follow our [bug_report.md](https://github.com/anitab-org/mentorship-backend/blob/develop/.github/ISSUE_TEMPLATE/bug_report.md) template.
+  Also please make sure you include steps to reproduce it and be patient we will get back to you as soon as we can.👀
```
Comment (isabelcosta) [inline .github/config.yml]:
```suggestion
  If you want to report a bug 🐞, please follow our [bug_report.md](https://github.com/anitab-org/mentorship-backend/blob/develop/.github/ISSUE_TEMPLATE/bug_report.md) template. Please make sure you include steps to reproduce it and be patient we will get back to you as soon as we can 👀
```

File: .github/config.yml [hunk 15]
```diff
@@ -0,0 +1,53 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  Hello there!👋 Welcome to the project!💖
+
+  Thank you and congrats 🎉 for opening your very first issue in this project.
+  AnitaB.org Open Source is an inclusive community, committed to creating a safe and positive environment 🌸 Please follow our [Code Of Conduct](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/code_of_conduct.md) 🙌
```
Comment (isabelcosta) [inline .github/config.yml]:
```suggestion
  AnitaB.org Open Source is an inclusive community, committed to creating a safe and positive environment 🌸  Please follow our [Code Of Conduct](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/code_of_conduct.md) 🙌 
  You can learn more about the community on [our website](https://anitab-org.github.io/) and [documentation repository](https://github.com/anitab-org/documentation#readme).
```

**Review by isabelcosta — APPROVED** [review]

**Review by vj-codes — APPROVED** [review]

**Review by epicadk — APPROVED** [review]

**Comment by isabelcosta** [comment]
Thank you so much @vj-codes and @epicadk for approving this 🙌🏾

**Pull request merged**